File: PYTHON/Fast.py
class FastWriter:
    def __init__(self, output_file=None):
        if output_file and os.path.exists(output_file):
            self.file = open(output_file, 'w')
        else:
            self.file = sys.stdout

    def spread_array(self, arr):
        self.file.write(' '.join(map(str, arr)) + '\n')

    def spread_list(self, lst):
        self.file.write(' '.join(map(str, lst)) + '\n')

    def println(self, value):
        self.file.write(str(value) + '\n')

    def print(self, value):
        self.file.write(str(value))

    def printf(self, format_string, *args):
        self.file.write(format_string % args)

    def close(self):
        if self.file != sys.stdout:
            self.file.close()


class FastReader:
    def __init__(self, input_file=None):
        if input_file and os.path.exists(input_file):
            self.file = open(input_file, 'r')
        else:
            self.file = sys.stdin
        self.tokens = []

    def next(self):
        while not self.tokens:
            if self.file:
                self.tokens = self.file.readline().split()
            else:
                self.tokens = input().split()
        return self.tokens.pop(0)

    def next_int(self):
        return int(self.next())

    def next_long(self):
        return int(self.next())

    def next_double(self):
        return float(self.next())

    def next_line(self):
        if self.file:
            return self.file.readline().strip()
        else:
            return input().strip()

File: JAVA/Special/Fast.java
package Special;
import java.io.*;
import java.util.*;

class FastWriter {
    PrintWriter pw;

    public FastWriter() {
        File file = new File("output.txt");
        if (file.exists()) {
            try {
                pw = new PrintWriter(file);
            } catch (FileNotFoundException e) {
                e.printStackTrace();
            }
        } else {
            pw = new PrintWriter(System.out);
        }
    }

    void spreadArray(int[] arr) {
        for (int i = 0; i < arr.length; i++) {
            print(arr[i] + " ");
        }
        pw.println();
    }

    void spreadArray(long[] arr) {
        for (int i = 0; i < arr.length; i++) {
            print(arr[i] + " ");
        }
        pw.println();
    }

    void spreadArray(double[] arr) {
        for (int i = 0; i < arr.length; i++) {
            print(arr[i] + " ");
        }
        pw.println();
    }

    void spreadList(List<?> list) {
        for (int i = 0; i < list.size(); i++) {
            print(list.get(i) + " ");
        }
        pw.println();
    }

    void println(Object o) {
        pw.println(o);
    }

    void print(Object o) {
        pw.print(o);
    }

    void printf(String format, Object... args) {
        pw.printf(format, args);
    }

    void close() {
        pw.close();
    }
}

class FastReader {
    BufferedReader br;
    StringTokenizer st;

    public FastReader() {
        File file = new File("input.txt");
        if (file.exists()) {
            try {
                br = new BufferedReader(new FileReader(file));
            } catch (FileNotFoundException e) {
                e.printStackTrace();
            }
        } else {
            br = new BufferedReader(new InputStreamReader(System.in));
        }
    }

    String next() {
        while (st == null || !st.hasMoreElements()) {
            try {
                st = new StringTokenizer(br.readLine());
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
        return st.nextToken();
    }

    int nextInt() {
        return Integer.parseInt(next());
    }

    long nextLong() {
        return Long.parseLong(next());
    }

    double nextDouble() {
        return Double.parseDouble(next());
    }

    String nextLine() {
        String str = "";
        try {
            str = br.readLine();
        } catch (IOException e) {
            e.printStackTrace();
        }
        return str;
    }
}